### PR TITLE
Add content and style labels

### PR DESCRIPTION
**Pull request by JuanFF**

### CODE REVIEW

**Comment by mrdonado** [comment]
`adding content` or `update repo` are very poor commit messages. Here I give you some feedback on how to write good commit messages: https://chris.beams.io/posts/git-commit/

File: assets/main.scss [hunk 1]
```diff
@@ -62,3 +62,124 @@ $on-laptop:        800px;
 a.site-title:hover{
   text-decoration: none;
 }
+
```
Comment (mrdonado) [inline assets/main.scss:65]: You don't need to rewrite the same rules again and again for different CSS classes. You can use something like:

´´´
.one-class, .another-class {
  height: 2rem;
  // whatever rules you need
}
´´´

**Review by mrdonado — COMMENTED** [review]
I'll include your changes now and later I'll add a commit refactoring a bit the CSS styles. Please, read the article about writing commits for the next time.

File: assets/main.scss (hunk 1 above)
Comment (JuanFF) [reply assets/main.scss:65]: I admit that my older commits on NLP were much more descriptive. The front-end world is completely new for me. You're right on the CSS code - I knew you would refactor it : )Thanks a lot. No more blog commits for now.

Comment (mrdonado) [reply assets/main.scss:65]: Sure, no problem at all :) just a bit of feedback here and there. You've done a great work with the blog. Let me know when you want to make it available online again.

**Pull request merged**